src/graphnet/models/model.py

```diff
@@ -110,6 +121,34 @@ def fit(
             self.warning("[ctrl+c] Exiting gracefully.")
             pass
 
+    def _create_default_callbacks(self, val_dataloader: DataLoader) -> List:
+        callbacks = [ProgressBar()]
+        callbacks = self._add_early_stopping(
+            val_dataloader=val_dataloader, callbacks=callbacks
+        )
+        return callbacks
+
+    def _add_early_stopping(
+        self, val_dataloader: DataLoader, callbacks: List
+    ) -> List:
+        if val_dataloader is not None:
+            has_es = False
+            assert isinstance(callbacks, list)
+            for callback in callbacks:
+                if isinstance(callback, EarlyStopping):
+                    has_es = True
+            if has_es is False:
+                callbacks.append(
+                    EarlyStopping(
+                        monitor="val_loss",
+                        patience=5,
+                    )
+                )
+                self.warn_once(
+                    "Got validation dataloader but no EarlyStopping callback. An EarlyStopping callback has been added automatically with patience=5 and monitor = 'val_loss'."
+                )
+        return callbacks
```

Comment:
Avoiding one level of nesting, wrapping a line to 79 characters, and just a wee bit of cosmetics.
```suggestion
        if val_dataloader is None:
            return callbacks
            
        has_early_stopping = False
        assert isinstance(callbacks, list)
        for callback in callbacks:
            if isinstance(callback, EarlyStopping):
                has_early_stopping = True
                
        if not has_early_stopping:
            callbacks.append(
                EarlyStopping(
                    monitor="val_loss",
                    patience=5,
                )
            )
            self.warn_once(
                "Got validation dataloader but no EarlyStopping callback. An "
                "EarlyStopping callback has been added automatically with "
                "patience=5 and monitor = 'val_loss'."
            )
        return callbacks
```